### PR TITLE
improved install/update of MSS

### LINKED ISSUE #957: update of MSS update description for new builds
if we have to create a new MSS build because of a dependency change it is safer for the user to get this new version by

```
mamba update mss
```

This `mamba update --all` seems not to take care for the build dependencies of the "build number" of mss  3.0.3            py39hf3d152e_1    conda-forge

### PR DESCRIPTION
**Pull request by ReimarBauer**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -47,33 +47,69 @@ Please add the channel conda-forge to your defaults::
...
++++++++
+
 You must install mss into a new environment to ensure the most recent
 versions for dependencies (On the Anaconda Prompt on Windows, you have to 
 leave out the 'source' here and below). ::
 
     $ conda create -n mssenv mamba
     $ conda activate mssenv
-    $ mamba install mss
+    $ (mssenv) mamba install mss
```
Comment (joernu76) [inline docs/installation.rst]: `(mssenv) $` ?
Same below...

File: docs/installation.rst [hunk 2]
```diff
@@ -47,33 +47,69 @@ Please add the channel conda-forge to your defaults::
...
 You need to reactivate after the installation once the environment to setup all needed enironment
 variables. ::
 
     $ conda deactivate
     $ conda activate mssenv
-    $ mss
+    $ (mssenv) mss
 
 
+update
```
Comment (joernu76) [inline docs/installation.rst:70]: I am unsure if we should not *only* recommend creating a new environment.

**Review by joernu76 — DISMISSED** [review]

File: docs/installation.rst (hunk 1 above)
Comment (ReimarBauer) [reply docs/installation.rst]: oh, yes, fixed

File: docs/installation.rst (hunk 2 above)
Comment (ReimarBauer) [reply docs/installation.rst:70]: at the moment we can update by mss this is solved too.

**Review by joernu76 — APPROVED** [review]

**Pull request merged**